grakn-engine/src/main/java/ai/grakn/engine/controller/ImportController.java

```diff
@@ -72,6 +72,9 @@
     private static final String MATCH_KEYWORD = "match";
 
     public ImportController(ClusterManager manager) {
+        if (manager==null) {
```

Comment:
This a precondition check, we don't use JAva's assert (not sure who does actually because it can be turned off), but I agree we need a utility class to check pre&post conditions. A bit like the junit assert, but for run-time sanity checks that make it easier to find bugs.